### PR TITLE
Close #25 install taxonworks on Win+WSL2+docker

### LINKED ISSUE #25: As a Developer I want instructions on how to setup taxonworks dev environment on Windows with WSL2

### PR DESCRIPTION
**Pull request by dimus**

### CODE REVIEW

File: development/docker-win/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    ```.bash
+    dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
+    ```
+    Restart your computer.
+
+* Setup WDL v2 as default
+
+    ```.bash
+    wsl --set-default-version 2
```
Comment (LocoDelAssembly) [inline development/docker-win/README.md:57]: When doing this step I was given the message "WSL 2 requires an update of your kernel component. For more information ..." -> https://aka.ms/wsl2kernel (Downloaded and installed WSL2 kernel from there)

Repeated `wsl --set-default-version 2` and this time it only shown a message indicating where I can get more info about key differences with WSL2 (https://aka.ms/wsl2)

File: development/docker-win/README.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+## Install Linux Distro
+
+There are several Linux distros available for WSL2, in our example we
+use Ubuntu 20.04.
+
+Go to `Microsoft Store` app, search for Ubuntu, install and launch it. You will
+see it appearing in a separate terminal this time. You can customize it right
+in the terminal, or you can close it, and reopen it in `Windows Terminal`,
+where it will appear now as an option.
+
```
Comment (LocoDelAssembly) [inline development/docker-win/README.md:69]: When launching Ubuntu 20.04 I got `WslRegisterDistribution failed with error: 0x80370102` and a message telling the virtual machine couldn't be started.

Did the following:
* Rebooted into BIOS setup
* Enabled `Secure Virtual Machine Mode` (Phenom II X4 955 CPU).
* Saved and booted Windows again
* Started up Ubuntu 20.04 app once more.

Now got 0x80370110: The VM or container with the specified identifier is not running.

Tried again and:
Asked to supply my UNIX username!

Takes away: Make sure hardware virtualization is enabled on BIOS, retry opening Ubuntu if it fails with 0x80370110 code.

File: development/docker-win/README.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+## Install Docker Desktop for Windows 10
+
+Download installer for Docker Desktop from the [Docker
+Hub](https://docs.docker.com/docker-for-windows/install/) and run the
+installation. Docker Desktop will also install hooks into WSL2, so you do not
+need to install Docker one more time on WSL2 Linux.
+
```
Comment (LocoDelAssembly) [inline development/docker-win/README.md:76]: After installing, the button to close the installer was "Close and logout". I was logged out, so had to re-open Ubuntu after logging back in.

File: development/docker-win/README.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+Now open Linux in Windows Terminal or its native terminal and try to run
+`docker` or `docker-compose` commands. If you get errors that tell you do not
+have enough privileges, run ``sudo vim /etc/group` and check that your account
+is added to docker group. For example I set my account as `dimus` and the line
+about docker group looks like this for me:
+
```
Comment (LocoDelAssembly) [inline development/docker-win/README.md:82]: I have used `sudo usermod -aG docker $USER` inside Ubuntu terminal. This is described at https://docs.docker.com/engine/install/linux-postinstall/ (creating the group was not necessary, it was already there)

**Review by LocoDelAssembly — COMMENTED** [review]
Preinstall comments. I'm in docker-compose build phase at this point so I think I won't get any more troubles.

**Comment by mjy** [comment]
@LocoDelAssembly @dimus should we merge this?

**Comment by dimus** [comment]
I am merging it

**Pull request merged**